### ISSUE TITLE
Trait/entity notes don't get populated through Bulk Upload Wizard

### ISSUE THREAD

**Issue by serbinsh** [PecanProject/bety#533]
In two different instances, I have noticed that properly formated .csv files uploaded through the Buld Upload Wizzard don't have their notes fields loaded into BETYdb?  

This is for data related to citations 

bety/citations/2000000017

and

bety/citations/2000000021

Both of which had a notes column populated with information that I was hoping to keep with the observations.

**Comment by serbinsh** [comment]
```

bety=> select notes from traits where entity_id = 2000000181;
 notes
-------





(5 rows)

```

**Comment by serbinsh** [comment]
```
bety=> select mean,variable_id from traits where entity_id = 2000000181;
 mean | variable_id
------+-------------
 9.59 |          15
  2.1 |          10
   48 |          11
 22.8 |          31
 2.19 |         566
(5 rows)

```

**Comment by serbinsh** [comment]
@gsrohde @dlebauer could we revisit this?  There are some datasets where I would like to keep the notes field as it contains information, such as plot number or lat/long for the measurement.  But as I mentioned, the notes field is recognized in the upload but after upload the notes are empty

**Comment by serbinsh** [comment]
Perhaps it wasnt the intention to keep the notes field?

**Comment by serbinsh** [comment]
For example


![screen shot 2017-11-01 at 7 45 09 am](https://user-images.githubusercontent.com/3067369/32273381-a8106fa2-bed8-11e7-9f20-f30b61bc79d3.png)

**Comment by gsrohde** [comment]
@serbinsh This was caused by a subtle but easy-to-fix bug introduced in March in release 4.15.  I will try to do a new release soon with this fixed.

**Comment by gsrohde** [comment]
P.S.  The notes always go into `traits.notes`, not `entities.notes`.  There is no way to populate the `entities.notes` column via the Bulk Upload Wizard.  This could possibly be changed if there is some agreement on how this should ideally work.

**Comment by gsrohde** [comment]
Pull request #549 fixes this.

**Comment by serbinsh** [comment]
Thanks @gsrohde!  It looks like the checks failed, do we need to modify anything before we can merge this fix?

**Comment by gsrohde** [comment]
@serbinsh If you mean the Travis checks, this is a known problem; see #544.

**Comment by serbinsh** [comment]
OK.  Does that mean we can merge your PR so that I can try the upload again?  @gsrohde @dlebauer

**Comment by gsrohde** [comment]
I'm waiting for one of the reviewers—@max-zilla, @robkooper, or @dlebauer—to approve the PR before I merge it.  I suppose you could check out the `bulk_upload_notes_fix` branch to your deployment copy in the meantime.

**Comment by gsrohde** [comment]
@serbinsh This is merged now, but I haven't done a new release yet.  If you don't want to wait, you could just pull the latest changes from master—there aren't any new database migrations since the last release.

**Comment by serbinsh** [comment]
Ok. Thanks!  I will likely pull the changes in master and then restart.

Sent from my iPhone

On Nov 6, 2017, at 2:58 PM, Scott Rohde <notifications@github.com<mailto:notifications@github.com>> wrote:


@serbinsh<https://github.com/serbinsh> This is merged now, but I haven't done a new release yet. If you don't want to wait, you could just pull the latest changes from master—there aren't any new database migrations since the last release.

**Comment by serbinsh** [comment]
Just to confirm, I will just pull the latest master branch and touch tmp/restart to load the updated code, correct?

**Comment by gsrohde** [comment]
`touch tmp/restart.txt`

**Comment by serbinsh** [comment]
Thanks

**Comment by gsrohde** [comment]
Fixed by release 4.20.  Closing.
